Sources/SourceKitLSP/Swift/CodeCompletion.swift

```diff
@@ -283,4 +298,56 @@ extension SwiftLanguageServer {
 
     return TextEdit(range: textEditRangeStart..<requestPosition, newText: newText)
   }
+  
+    /// Must be called on `queue`.
```

Comment:
If this must be called on `queue`, could you add a dispatch precondition checking it. E.g. https://github.com/apple/sourcekit-lsp/blob/d29d1254a4bf19e91d04eec441c56b6142b2a4c6/Sources/SourceKitLSP/Swift/SwiftLanguageServer.swift#L170

Reply:
![image](https://user-images.githubusercontent.com/78330558/154602107-8ff18ae5-848e-4245-9965-9e3e71a19fb5.jpeg)